win32ss/user/user32/windows/input.c

```diff
@@ -638,12 +638,10 @@ LoadKeyboardLayoutA(LPCSTR pszKLID,
     return LoadKeyboardLayoutW(wszKLID, Flags);
 }
 
-/*
- * @implemented
- */
-HKL WINAPI
-LoadKeyboardLayoutW(LPCWSTR pwszKLID,
-                    UINT Flags)
+/* Win: LoadKeyboardLayoutWorker */
+HKL APIENTRY
+IntLoadKeyboardLayout(DWORD unknown1, LPCWSTR pwszKLID, LPCWSTR unknown3, UINT Flags,
```

Comment:
Please take the opportunity, for any new (or fully refactored) function, to use SAL annotations and parameters on their own lines.

Reply:
Added SAL annotations. OK.